tests/commands/test_cmd_config.py

```diff
@@ -0,0 +1,73 @@
+# third-party imports
+from click.testing import CliRunner
+
+# local imports
+from shpkpr.cli import cli
+
+
+def test_no_args():
+    runner = CliRunner()
+    result = runner.invoke(cli, ['config'])
+
+    assert result.exit_code == 0
+    assert 'Usage:' in result.output
```

Comment:
Yeah they're very similar atm. The plan for these tests was to treat them almost like unit tests, mocking out marathon/mesos as required and using them to check edge cases, various combos of options, etc.

The integration tests are designed to be run much less frequently, against real marathon and mesos instances to verify operation against real servers.

The two can probably share some common setup/fixtures, but I'd like to keep both types of test as I feel they'll be useful for different reasons.